static/js/hotkey.js

```diff
@@ -781,16 +781,14 @@ export function process_hotkey(e, hotkey) {
         case "star_deprecated":
```

Comment:
Oh, should we add another commit to move this below?  I think the `*` shortcut only ever worked with a message selected, so doesn't belong here.